symphony-bdk-legacy/symphony-bdk-bot-sdk-java/src/test/java/com/symphony/bdk/bot/sdk/sse/SsePublisherRouterImplTest.java

```diff
@@ -0,0 +1,129 @@
+package com.symphony.bdk.bot.sdk.sse;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import com.symphony.bdk.bot.sdk.sse.config.SseSubscriberProps;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.springframework.web.servlet.mvc.method.annotation.SseEmitter;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class SsePublisherRouterImplTest {
+
+  private SsePublisherRouterImpl ssePublisherRouterImpl;
+  private SsePublisher<?> ssePublisher = Mockito.mock(SsePublisher.class);
+
+  @Before
+  public void initSsePublisher() {
+    this.ssePublisherRouterImpl = new SsePublisherRouterImpl();
+  }
+
+  @Test
+  public void testRegister() {
+    final List<String> stringsList = this.initStringsList();
+    Mockito.when(this.ssePublisher.getEventTypes()).thenReturn(stringsList);
+
+    this.ssePublisherRouterImpl.register(this.ssePublisher);
+    // We can't verify that we have added well the ssePublisher because the list is private and there no getter
+  }
+
+  @Test
+  public void testFindPublishers() {
+    final List<String> eventTypes = this.initStringsList();
+    Mockito.when(this.ssePublisher.getEventTypes()).thenReturn(eventTypes);
+
+    // We have to register at least 2 ssePublishers in the list with a common event type to get a non-empty list
+    this.ssePublisherRouterImpl.register(this.ssePublisher);
+    this.ssePublisherRouterImpl.register(this.ssePublisher);
+
+    final List<SsePublisher<?>> publishers = this.ssePublisherRouterImpl.findPublishers(eventTypes);
+    assertNotNull(publishers);
+    assertEquals(2, publishers.size());
+    for (final SsePublisher<?> publisher : publishers) {
+      assertFalse(publisher.getEventTypes().isEmpty());
+
+      final List<String> eventTypesPublisher = publisher.getEventTypes();
+      assertEquals(3, eventTypesPublisher.size());
```

Comment:
`assertEquals(initStringsList(), publisher.getEventTypes())` or `assertEquals(Arrays.asList("string1", "string2", "string3"), publisher.getEventTypes())` would do it and is much shorter.